src/sentry.ts

```diff
@@ -0,0 +1,18 @@
+import * as Sentry from '@sentry/browser';
+import { ocmClient } from './api/axiosClient';
+
+export enum SEVERITY {
+  ERROR = 'error',
+  WARN = 'warning',
+}
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+export const captureException = (error, message?: string, severity: SEVERITY = SEVERITY.ERROR) => {
+  if (ocmClient) {
+    message && Sentry.captureMessage(message, SEVERITY[severity]);
```

Comment:
Same comment as the structured parameters to Sentry.
maybe we can send additional parameters instead of string only.